### DISCUSSION TITLE
Is DATAS GC still considered experimental?

### DISCUSSION THREAD

**Discussion by daltonks** [dotnet/runtime#95993] [Q&A] [answered]
I'm trying to see if I can use it in a production environment 🙂
I see that it is used by default for AOT applications, so maybe it's no longer experimental?

**Comment by jkotas** [comment, marked as answer]
DATAS is supported in production environment. You can give it a try.

In .NET 8, it is not fine-tuned well enough to make us comfortable with turning it on everywhere by default. If you run into performance issues with it, let us know.

**Reply by daltonks** [reply]
Thank you!
As for it not being fine-tuned enough, does that just mean the feature doesn't perform as well as you'd like under some situations?

**Reply by jkotas** [reply]
Right
